api.go

```diff
@@ -77,16 +78,28 @@ func (e *ErrorResponse) Temporary() bool {
 }
 
 func (api *API) request(endpoint string, method string, params map[string]interface{}, body *bytes.Buffer) (result *bytes.Buffer, status int, err error) {
+	bucketLimit, err := strconv.Atoi(os.Getenv("BUCKET_LIMIT"))
```

Comment:
I think instead of getting these things through the environment every time, we could create a `Config` structure with a package-level variable to house these values. That way we could at least set sensible defaults so people won't shoot themselves in the foot by forgetting to set these env vars.